en-US/win10/samples/DriverLab4.md

```diff
@@ -17,27 +17,26 @@ This section describes the steps required to configure your target device for dr
 * Connect to your target device via Powershell as described in the instructions [here]({{site.baseurl}}/{{page.lang}}/win10/samples/PowerShell.htm).
 * For current release there are a few provisioning steps you need to do.  These steps will be automated in the future.
 * Now configure your target device so that it can interact with Visual Studio.  Through the PowerShell window enter the commands listed below.
-		
-		cd c:\windows\system32
-		.\sc.exe config sshsvc start=disabled
-		.\sc.exe stop sshsvc
-		.\sc.exe config TestSirepSvc start=auto
-		.\sc.exe start TestSirepSvc
-		.\reg.exe add HKLM\SOFTWARE\Microsoft\SystemCertificates\Root\Certificates\8a334aa8052dd244a647306a76b8178fa215f344
-		
+        
+        cd c:\windows\system32
+        .\sc.exe config sshsvc start=disabled
+        .\sc.exe stop sshsvc
+        .\sc.exe config TestSirepSvc start=auto
+        .\sc.exe start TestSirepSvc
+        .\reg.exe add HKLM\SOFTWARE\Microsoft\SystemCertificates\Root\Certificates\8a334aa8052dd244a647306a76b8178fa215f344
+        
 Your target device is now configured for driver development using Visual Studio.
 
 ###On the development computer
 
 * Install the Windows Driver Kit on your development machine by following the instructions [here](https://msdn.microsoft.com/en-US/windows/hardware/dn913721(v=vs.8.5).aspx).  You will need to install the SDK and WDK.
-
 * Install the certificates so that the driver is signed correctly and can run on your target device.  In later steps, when you sign your driver make sure to use the 'WDKTestCert'. From a command prompt execute the commands listed below:
 
-		cd c:\Program Files (x86)\Windows Kits\10\Tools\bin\i386
-		Set WPDKContentRoot=c:\Program Files (x86)\Windows Kits\10		
-		InstallOEMCerts.cmd
+        cd c:\Program Files (x86)\Windows Kits\10\Tools\bin\i386
+        Set WPDKContentRoot=c:\Program Files (x86)\Windows Kits\10      
+        InstallOEMCerts.cmd
 
- Add your target device to Visual Studio
+* Add your target device to Visual Studio
```

Comment:
Same as Matt's comment below. This looks more like a sub-header than a bullet point.